app/app.go

```diff
@@ -88,6 +90,29 @@ func Stack(issuer weave.Address, minFee coin.Coin) weave.Handler {
 	return Chain(authFn, minFee).WithHandler(Router(authFn, issuer))
 }
 
+// CronStack wires up a standard router with a cron specific decorator chain.
+// This can be passed into BaseApp.
+// Cron stack configuration is a subset of the main stack. It is using the same
```

Comment:
Cron stack does not have to be a subset of the main stack. It all depends on what messages the cron should have access to.
Cron messages can be handled by different set of handlers and different middleware configuration. That allows you to for example not charge any fee for their execution or disable/enable certain messages and therefore functionality.


This is not a wrong comment. I want to make sure that you understand why we did it like this in bnsd.

Reply:
I see :+1: I will explain this in https://github.com/iov-one/docs/issues/50